### DISCUSSION TITLE
为什么可以直接 `@import 'common'`？

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#3941] [提问 / Q&A] [answered]
为什么它可以不用相对路径？这个事情是什么插件干的吗？我的 WebStorm 无法正确识别 common 这个路径，只能使用相对路径。

**Comment by the1812** [comment, marked as answer]
https://github.com/the1812/Bilibili-Evolved/blob/b9a1a89d88351f394f66ff8f973c3ba6f55464e0/webpack/loaders/style-loaders.ts#L19-L24
